src/docs/reference/modules/Templates/README.md

```diff
@@ -76,7 +76,7 @@ For instance, when a content item is displayed in a list, the `Summary` display
 
 This template is called when displaying a content item with a specific Id.
 
-Note: This template was supported on Orcard Core version 1.X and will probably be removed from future versions. It is strongly recommended to use the corresponding template that utilises the ContentItemId.
+Note: This template was supported on Orchard version 1.X and will probably be removed from future versions. It is strongly recommended to use the corresponding template that utilizes the ContentItemId.
```

Comment:
> and will probably be removed from future versions

Your source?

Reply:
@sebastienros do you remember if this is still supported in Orchard CMS?

/cc @BenedekFarkas

Reply:
@hishamco I think it's just quoting the issue, which states that there is no reason to use Id as it's not stable (I agree). So we could remove that later but not right now as we need to provide a transition window (let users migrate to Id). So maybe file an issue to remove this and assign to the next release? (not a .x one).

Reply:
@dmourtzoukos Don't we have alias and route already generating these alternates? That could be a nicer option in most cases. Not sure how many items you would have that are not accessible in either way. But I have nothing against this PR, if not that it's yet another shape that allocates quite a bit, but we'll optimize this eventually.

Reply:
@sebastienros there is no documentation about alternates for Alias or Autoroute (and could not find any shape table provider in code). My concern here has to do with localized items, with any of the three approaches -Alias, Route and ContentItemId-.
I mean, how could somebody override the template for a localization set? Defining templates per ContentItemId and using a shared liquid? Maybe is better to also introduce alternates per LocalizationSet?

Reply:
As part of the Orchard Harvest Hackathon, I'm adding alternates for the alias and the routes. The issue specific to this is #2866 and the pull request is #16722